Image3dAPI/Image3dAPI.vcxproj

```diff
@@ -85,13 +85,16 @@
     <Midl>
       <OutputDirectory>$(IntDir)</OutputDirectory>
       <HeaderFileName>%(Filename).h</HeaderFileName>
-      <TypeLibraryName>$(ProjectName).tlb</TypeLibraryName>
+      <TypeLibraryName>$(OutDir)$(ProjectName).tlb</TypeLibraryName>
     </Midl>
     <PostBuildEvent>
-      <Command>xcopy /Y /D "$(IntDir)$(TargetName).tlb"  "$(SolutionDir)$(Platform)"
-TlbImp.exe /machine:x86 "$(IntDir)$(TargetName).tlb" /out:$(SolutionDir)$(Platform)\$(TargetName).dll
+      <Command>TlbImp.exe /machine:x86 "$(OutDir)$(TargetName).tlb" /out:$(OutDir)$(TargetName).dll
+:: Copy interface definitions to shared project dir
 xcopy /Y /D "$(IntDir)*.h"  "$(ProjectDir)"
-xcopy /Y /D "$(IntDir)*.c"  "$(ProjectDir)"</Command>
+xcopy /Y /D "$(IntDir)*.c"  "$(ProjectDir)"
```

Comment:
Please make the build fail if any of the xcopy calls fail by appending || exit /b 1

Reply:
Agree. Fixed.